src/smt/smt_engine.cpp

```diff
@@ -4215,9 +4202,13 @@ void SmtEnginePrivate::processAssertions() {
     Trace("smt-proc") << "SmtEnginePrivate::processAssertions() : pre-static-learning" << endl;
     // Perform static learning
     Chat() << "doing static learning..." << endl;
```

Comment:
The `Chat()` and `Trace()` statements can be removed, the PreprocessingPass class is taking care of communicating that.